website_docs/_index.md

```diff
@@ -27,7 +27,7 @@ is as follows:
 | -------   | ------- | ------------------- |
 | Stable    | Alpha   | Not Yet Implemented |
 
-The current release can be found [here](https://github.com/open-telemetry/opentelemetry-python/releases)
```

Comment:
Why is `The current release can be found` removed?

Reply:
It is being replaced by text that is share among several SIGs:

> {{% latest_release "python" /%}}

This is the shortcode: https://github.com/open-telemetry/opentelemetry.io/blob/main/layouts/shortcodes/latest_release.md

Reply:
@ocelotl - can this be merged soon?  This is the last SIG to do so before I can wrap up this feature, thanks.

Reply:
Sure, approving :v: thanks @chalin !